pandas/core/indexes/timedeltas.py

```diff
@@ -383,6 +383,34 @@ def join(self, other, how='left', level=None, return_indexers=False,
                           return_indexers=return_indexers,
                           sort=sort)
 
+    def intersection(self, other, sort=False):
```

Comment:
hmm, we have specialized do-strings for all of these? Is this the same way we handle union? 

cc @reidy-p

Reply:
@jreback I am not sure what do-strings are?

Reply:
doc-string

Reply:
Yeah I think we have specialised doc-strings for the set operations that override the base class

Reply:
Yes there is a specialised doc-string in /pandas/core/indexes/timedeltas that overwrites the docstring for intersection.